account_invoice_product_supplier_price_update/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Chafique DELLI @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Account Invoice Product Supplier Price Update',
+    'summary': 'In the supplier invoice, automatically update all products '
+               'whose unit price on the line is different from '
+               'the supplier price',
+    'version': '8.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'website': 'http://akretion.com',
+    'author': 'Akretion, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'depends': [
+        'account',
+        'purchase',
+        'product_variant_supplierinfo',
```

Comment:
This module and the following are not real dependencies. Just conveniences, but the module will work the same, except for that features.